app/scripts/services/resourceAlerts.js

```diff
@@ -100,6 +101,33 @@ angular.module("openshiftConsole")
       }
     };
 
+    // deploymentConfig, k8s deployment
+    var getPausedDeploymentAlerts = function(alerts, deployment) {
```

Comment:
Here you take in the `alerts` object as a parameter, but in `getDeploymentStatusAlerts`, it's a return value. Would be good to be consistent